src/main.rs

```diff
@@ -195,8 +220,15 @@ fn main() {
 
     let (metrics_controller, metrics_sink) = metrics::build();
 
+    let json = match &opts.log_format {
+        Some(LogFormat::Text) => false,
+        Some(LogFormat::Json) => true,
+        None => false,
+    };
+
```

Comment:
Can't this simply be `let json = opts.log_format == Some(LogFormat::Json);`?
Or, to make the default explicit, `let json = opts.log_format.unwrap_or(LogFormat::Text) == LogFormat::Json;`